packages/component/src/Styles/defaultStyleOptions.js

```diff
@@ -187,7 +187,11 @@ const DEFAULT_OPTIONS = {
   toastSuccessBackgroundColor: '#DFF6DD',
   toastSuccessColor: '#107C10',
   toastWarnBackgroundColor: '#FFF4CE',
-  toastWarnColor: '#3B3A39'
+  toastWarnColor: '#3B3A39',
+
+  // Emoji
+  emojiAutocorrect: false,
```

Comment:
Why do we need a boolean? The user could just make the `emojiList` falsy to turn off this feature, and this approach is more intuitive, because it reduce "2 ways to turn off emoji" into "a single way to turn off emoji."

Reply:
I wanted the boolean so that the user can enable or disable, and then they only need to provide an emoji list if they want to. This comment and:
![image](https://user-images.githubusercontent.com/14900841/87755979-e8b24e00-c7bc-11ea-8d93-f9438cf70e60.png)
seem to conflict. Could you clarify?

If `emojiList` is set to falsy, then user would always need to provide a list. Which isn't necessarily incorrect, but I thought it would be more important for the emoji feature to work 'out of the box' if enabled.

Reply:
To disable emoji:
  
- Pass `false` to `emojiAutocorrect` or do nothing (default is `false`)
- Pass empty array to the emoji table

To enable emoji:
 
- Pass `true` to `emojiAutocorrect`
- And optionally, pass an array to the emoji table

Since there are multiple ways to do one thing, there can be conflicting options between them. What if:
 
- Pass an array to emoji table and did not set `emojiAutocorrect`?
- Pass an empty array to the emoji table and set `emojiAutocorrect` to `true`?

With two options and they are conflicting, there are no single source of truths.
 
Ultimately, we have 3 user stories to achieve:

- Disable emoji
- Enable emoji and use default mapping
- Enable emoji and use custom mapping
  - Customization can be:
    - Adding rules from default set
    - Removing rules from default set
    - Replacing whole set

(I am okay to leave the "add/remove/replace rules", since the default list is short and easily copyable.)
 
You can use a single options to reduce confusions.

- `emojiList: false` (disable emoji)
- `emojiList: true` (enable default emoji)
- `emojiList: { ':)': '' }`
  - This always replace the whole set.